src/Item.cpp

```diff
@@ -1,4 +1,4 @@
-#include "Item.hpp"
+#include "../include/Item.hpp"
```

Comment:
Why where these changed?  The makefile has the -I flag which specifies the include directory, so it shouldn't be needed to hardcode the path.

Reply:
I had set that when the makefile wasn't there and forgot I had set them.